### PR TITLE
Issue 4

### PR DESCRIPTION
**Pull request by HenrikOssipoff**

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [settings]
 known_first_party=altapay
+known_third_party=requests,six,xmltodict
```
Comment (HenrikOssipoff) [inline .isort.cfg]: xmltodict needs to be removed

File: altapay/api.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
+from __future__ import absolute_import, unicode_literals
+
+import logging
+import os
+import platform
+import ssl
+from xml.etree import ElementTree
```
Comment (DevDapo) [inline altapay/api.py]: Why not use lxml? :8ball:

Comment (HenrikOssipoff) [inline altapay/api.py]: I don't want to depend on another external library for a third party package, and the documents are typically rather small, so the gain is insignificant.

Comment (DevDapo) [inline altapay/api.py]: :+1:

**Comment by DevDapo** [comment]
LGTM

**Pull request merged**